servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/SubscriptionDescriptionSerializer.java

```diff
@@ -245,6 +242,8 @@ private static SubscriptionDescription parseFromEntry(String topicName, Node xEn
                                             sd.forwardDeadLetteredMessagesTo = fwdDlq.getNodeValue();
                                         }
                                         break;
+                                    default:
```

Comment:
[SpotBugs-P2]: Switch statement found in com.microsoft.azure.servicebus.management.SubscriptionDescriptionSerializer.parseFromEntry(String, Node) where default case is missing